nwind-capi/include/nwind_capi.h

```diff
@@ -0,0 +1,84 @@
+/* SPDX-License-Identifier: MIT */
+
+#ifndef NWIND_CAPI_H_
+#define NWIND_CAPI_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Opaque handle for a local address space */
+typedef struct nwind_address_space_ nwind_address_space;
+
+/* Opaque handle for a local unwind context */
+typedef struct nwind_unwind_context_ nwind_unwind_context;
+
+/**
+ * Create a local address space to use for unwinding.
+ *
+ * This type can be reused across threads, as long as the address space stays the same.
+ * When dlopen is called or the address space is otherwise mutated, the address space
+ * needs to be reloaded.
+ *
+ * @param use_shadow_stack set to 1 when you want to use the shadow stack.
+ *
+ * @sa nwind_free_address_space, nwind_reload_address_space
+ * @sa nwind_create_unwind_context, nwind_backtrace
+ */
+nwind_address_space *nwind_create_address_space(int use_shadow_stack);
```

Comment:
can it be a non-opaque one instead? allocating memory just to pass a single bool flag feels like overkill to me :)

Reply:
Won't be ABI forward compatible, but yeah, that's fine too. (So there probably should be a function to return a default preinitialized config struct, since it's not the 1970s and we shouldn't `memset` to zero anymore. :P)